### PR TITLE
Use brighter red and green colors.  The dark red and green are difficult to read.

### PR DESCRIPTION
**Pull request by haroldalmon**
Use brighter colors.  The default DarkRed and Darkgreen are difficult to read.

### CODE REVIEW

**Comment by dahlbyk** [comment]
So I've been stubbornly pushing back on this (see #72) for a long time, but I think I've finally convinced myself that this needs to be fixed. However, I'd still prefer to align with the `git status` default if the console isn't using a ridiculous blue background. So...what if these settings had a bit of smarts to detect if we're in a default PowerShell host (with a "blue" `DarkMagenta` background)?

```powershell
BeforeIndexForegroundColor= if ($Host.UI.RawUI.BackgroundColor -eq [ConsoleColor]::DarkMagenta) { [ConsoleColor]::Green } else { [ConsoleColor]
::DarkGreen }
```

Is that a reasonable compromise?

**Comment by haroldalmon** [comment]
Yes, but I believe you need to set the IndexForegroundColor also. I believe the BeforeIndexForegroundColor is the color of a default empty string. The resulting color is very good.

Also, I recommend you set the WorkingForegroundColor and WorkingForegroundColor to Red instead of DarkRed. I rate DarkRed as poor. Red as fair. I tried this but did not like the result. The Red is still too dark. I think the Git default should at least be Red. If people change the Git defaults to Green and Red, at least the screen is readable.

For my own PC, I gave up on the standard colors and used cyan and yellow instead of DarkRed. It work be nice if Git used Yellow for untracked and maybe unmodified files. Maybe I'll ask the Git team to consider changing the default DarkRed to Yellow.

Anyway, go with the change you suggested and consider doing the same for the two DarkRed I mentioned above.

**Comment by dahlbyk** [comment]
I was just using that as an example - the same pattern would be used on all the values you changed here.

As for changing the Git defaults, that's already configurable - see https://github.com/dahlbyk/posh-git/issues/72#issuecomment-20050873.

**Comment by haroldalmon** [comment]
I think you should implement your idea. Users are going to like the improvement.

**Comment by dahlbyk** [comment]
Care to take a crack at it and push another commit to this PR?

**Comment by haroldalmon** [comment]
Sure, will code and test on the weekend.

**Comment by haroldalmon** [comment]
I had to change your sample code and decided to add some other properties so that the original properties were not dead code.  The original colors are used if the screen background is not DarkMagenta I tested this change and the code seems to work.  

I pushed the changes to my GitHub account.  Do I need to do anything else to get the code to you?  

Do you want any more changes?

File: GitPrompt.ps1 [hunk 1]
```diff
@@ -167,6 +171,15 @@ function Write-GitStatus($status) {
 if(!(Test-Path Variable:Global:VcsPromptStatuses)) {
     $Global:VcsPromptStatuses = @()
 }
+$s = $global:GitPromptSettings
+
+# Override some of the normal colors if the background color is set to the default DarkMagenta.
+if ($Host.UI.RawUI.BackgroundColor -eq [ConsoleColor]::DarkMagenta) { $s.BeforeIndexForegroundColor = $s.BeforeIndexForegroundBrightColor }
```
Comment (dahlbyk) [inline GitPrompt.ps1]: This approach works for me. Can you combine these into a single `if` block?

**Comment by dahlbyk** [comment]
> I pushed the changes to my GitHub account. Do I need to do anything else to get the code to you?

Nope, your update worked perfectly. Note the new commits that showed up before your comment (and on the Commits tab).

I think a single `if` would be easier to read, otherwise this seems good to :ship:.

File: GitPrompt.ps1 (hunk 1 above)
Comment (haroldalmon) [inline GitPrompt.ps1]: I collapsed the 4 ifs into one block and pushed the change to GitHub.  I tested the change.

**Comment by dahlbyk** [comment]
:sparkles: Thanks!

**Comment by haroldalmon** [comment]
Thanks for walking me through the process. I saw Matthew J McCullough talk about pull requests in a youtube video a few months ago and though I would try one.

**Comment by dahlbyk** [comment]
You bet. If you're looking for other contribution opportunities, check out http://up-for-grabs.net/.

**Pull request merged**